TestFoundation/TestNSURL.swift

```diff
@@ -212,13 +211,8 @@ class TestNSURL : XCTestCase {
                 XCTFail()
             }
             if let url = url {
-
-                // TODO: NSURL.standardizedURL isn't implemented yet.
-                var modifiedExpectedNSResult = expectedNSResult as! [String: Any]
-                modifiedExpectedNSResult["standardizedURL"] = nil
-                
                 let results = generateResults(url, pathComponent: inPathComponent, pathExtension: inPathExtension)
-                let (isEqual, differences) = compareResults(url, expected: modifiedExpectedNSResult, got: results)
+                let (isEqual, differences) = compareResults(url, expected: expectedNSResult as! [String: Any], got: results)
```

Comment:
linux probably won't like this; most likely it will have to be a mapped result (have not yet tested, but I will follow up with results on tests shortly)